pkgs/top-level/all-packages.nix

```
@@ -21205,11 +21205,14 @@ in
     inherit (darwin) libobjc;
     inherit (darwin) libresolv;
     guile = guile_2_0;
+    pythonPackages = python3Packages;
   };
 
   weechat = wrapWeechat weechat-unwrapped { };
 
-  weechatScripts = dontRecurseIntoAttrs (callPackage ../applications/networking/irc/weechat/scripts { });
+  weechatScripts = dontRecurseIntoAttrs (callPackage ../applications/networking/irc/weechat/scripts {
```

dontRecurseIntoAttrs is now the default, I think the implementation now is just the id function `drv: drv`
```suggestion
  weechatScripts =  callPackage ../applications/networking/irc/weechat/scripts {
```

---

yep, had to check https://github.com/NixOS/nixpkgs/blob/5e86891baa57ba4e8aace81c0f24399df3082d37/pkgs/top-level/all-packages.nix#L75

---

In fact it's always been the default, `dontRecurseIntoAttrs` mainly serves a documentation purpose. To quote the comment above the function:

```
  # This is intended to be the reverse of recurseIntoAttrs, as it is
  # defined now it exists mainly for documentation purposes, but you
  # can also override this with recurseIntoAttrs to recurseInto all
  # the Attrs which is useful for testing massive changes. Ideally,
  # every package subset not marked with recurseIntoAttrs should be
  # marked with this.
```

---

ah, if it has a purpose, then I'm fine with leaving it in :)